.github/workflows/universalnrp-test.yml

```diff
@@ -183,18 +173,94 @@ jobs:
         if: success() || failure()
         with:
           name: ${{ steps.normalize.outputs.name }}_report
-          path: '${{ steps.normalize.outputs.PolicyPackageDir }}/*testResults.xml'
+          path: '${{ steps.normalize.outputs.dir }}/*testResults.xml'
 
       - uses: actions/upload-artifact@v4
         if: success() || failure()
         with:
           name: ${{ steps.normalize.outputs.name }}_logs
           path: osconfig-logs/osconfig*
 
+  module-test:
```

Comment:
Why are we moving `Module Test` into this workflow? Isn't it better to have it broken out to a different workflow given it also has different targets?

Reply:
This module test run uses the same "universal" shared build artifacts as the MC tests and only tests the security baseline recipe across all the target distros.

Reply:
Ahmed, we discussed about this and got agreement (you were present).